modules/nf-core/pbbam/pbmerge/tests/main.nf.test

```diff
@@ -0,0 +1,36 @@
+nextflow_process {
+
+    name "Test Process PBBAM_PBMERGE"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "PBBAM_PBMERGE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pbbam"
+    tag "pbbam/pbmerge"
+
+    test("Main test") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [
+                        file(params.test_data['homo_sapiens']['pacbio']['cluster']   , checkIfExists: true),
```

Comment:
I think this might fail because you give two paths? But maybe that is not the issue.

Reply:
Having two files/paths is expected here. pbmerge merges bam files into one.